versions.yaml

```diff
@@ -168,16 +168,16 @@ assets:
 
   kernel:
     description: "Linux kernel optimised for virtual machines"
-    url: "https://cdn.kernel.org/pub/linux/kernel/v5.x/"
-    version: "v5.19.2"
+    url: "https://cdn.kernel.org/pub/linux/kernel/v6.x/"
+    version: "v6.1.38"
     sev:
       description: "Linux kernel that supports SEV and SNP"
-      url: "https://cdn.kernel.org/pub/linux/kernel/v5.x/"
-      version: "v5.19.2"
+      url: "https://cdn.kernel.org/pub/linux/kernel/v6.x/"
+      version: "v6.1.38"
```

Comment:
Based on @zvonkok comment, let's not update this one yet.

Reply:
pulled this back down to v5.19.2